test/sol6/kyberIntegration.js

```diff
@@ -218,10 +218,6 @@ contract('Proxy + Network + MatchingEngine + FeeHandler + Staking + DAO integrat
         await stakingContract.deposit(initMikeStake, {from: mike});
     };
 
-    beforeEach("running before each test", async() => {
-        await updateCurrentBlockAndTimestamp();
-    });
-
     const tradeAndCheckDataChangesAsExpected = async(epoch, expectedNetworkFee, expectedReward, expectedRebate, logGasMsg) => {
```

Comment:
please move this function to bottom of file. where we have helper functions.

Reply:
done